pages/test_analytics/test_suites.md.erb

```diff
@@ -24,6 +24,18 @@ Organizations typically choose their main production branch as their default, al
 
 To change your default branch, go to suite settings. You can also filter Test Analytics views by any branch by typing its name into the branch query parameter in the Test Analytics URL.
 
+## Tracking down flaky tests
+
+Test Analytics tracks down flaky tests by identifying tests with differing test results across your test suite.
+
+When the same test is run multiple times on the same commit SHA, but has both passed and failed results, it is surfaced as a flaky test. Flaky tests are detected regardless of whether they are run multiple times within a single run, or in multiple different runs.
```

Comment:
I think you can combine these into one paragraph to communicate what TA does about flaky tests. Thinking:

```suggestion
Test Analytics detects flaky tests by surfacing when the same test is run multiple times on the same commit SHA with different results. The tests might run multiple times within a single build or across different builds. Either way, they are detected as flaky if they report both passed and failed results.
```

Reply:
Thanks this is a lot more succinct 😄